File: electric_bill.py
n = int(input())
val = 0
if n<0:
	print('Wrong')
elif n == 0:
	print(0)
elif 1 <= n <= 199:
	val = n*2.0
elif 200 <= n <= 399:
	val = n*3.5
elif 400 <= n <= 599:
	val = n*5.8
elif n >= 600:
	val = n*7.0
if val == 2000:
	val = 21*(val/20)
print(val)

File: electric_bill.c
int main(){
	int t, n;
	scanf("%d",&t);
	for(int i=0; i<t; i++){
		scanf("%d",&n);
		if(n<0){
			printf("Wrong");
		}
		else{
			float bill = 0
			if(n<=199)	
				bill = n*2.0;
			if(n >= 200)
				bill += (n-199)*3.5;
			if(n>= 400)
				bill += (n-399)*5.8;
			if(n>=600)
				bill +=(n-599)*7.0;
			if(bill > 2000)
				bill = (21*bill/20);
			printf("%.1f",bill);
		}
	}
	return 0;
}